### ISSUE TITLE
Chat messages not sorting properly

### ISSUE THREAD

**Issue by klaw23** [firebase/firechat-ios#3]
It looks like this code is using the same sorting as the firebase web interface which isn't always consistent.

-J4... < -J-...
![](https://dl.dropbox.com/s/scu0trfluhtwvxq/Screen%20Shot%202013-10-02%20at%2010.19.25%20AM.png)

-J-... < -J0...
![](https://dl.dropbox.com/s/ef4xxu2yn7sk601/Screen%20Shot%202013-10-02%20at%2010.19.38%20AM.png)

Do we need to use `- observeEventType:andPreviousSiblingNameWithBlock:` to keep the messages in order even if we're not using priority?

In other words, is this a bug in the firechat-ios client or the firechat backend?

Thanks!

**Comment by robertdimarco** [comment]
Closing since there hasn't been any activity on here in ages, but @klaw23 don't hesitate to send us a PR if you're interested in incorporating your changes into this lib!
